docs/glossary.rst

```diff
@@ -478,10 +478,42 @@ defined role within the organization (see our reference on :ref:`user roles
 
     :ref:`User roles <user-roles>`
 
+.. _gloss-version:
+
+Versions
+--------
+
+CrateDB uses a versioning system with three levels of versioning: major
```

Comment:
you could optionally make a reference to the [Semver](https://semver.org/) versioning system here that might already be familiar to some users.

So something like 

```suggestion
CrateDB uses a semantic versioning system, known as [Semver](https://semver.org/).  This system comprises  three levels of versioning: major
```